.github/workflows/publish.yml

```diff
@@ -12,15 +12,15 @@ concurrency:
 jobs:
   publish-crates:
     if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Check out repository
         uses: actions/checkout@v3
 
       - name: Setup rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.63
+          toolchain: 1.64
```

Comment:
If you update the toolchain, you must align this version everywhere, i.e.: `Dockerfile` and the `Cargo.toml`